### PR TITLE
[14.0][MIG] Migration to version 14.0: graphql_base

### PR DESCRIPTION
**Pull request by qgroulard**

### CODE REVIEW

File: graphql_base/__manifest__.py [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
         Base GraphQL/GraphiQL controller""",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
     "license": "LGPL-3",
     "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/rest-framework",
     "depends": ["base"],
     "data": ["views/graphiql.xml"],
-    "external_dependencies": {"python": ["graphene", "graphql-server-core"]},
+    "external_dependencies": {"python": ["graphene", "graphql-server", "promise"]},
```
Comment (sbidoul) [inline graphql_base/__manifest__.py]: As discussed, `promise` is used only by `graphene.test.Client` so this dependency is best declared in `test-requirements.txt`.

File: graphql_base/__manifest__.py [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
     "development_status": "Beta",
```
Comment (sbidoul) [inline graphql_base/__manifest__.py]: I think we can move this to `Production/Stable`

**Review by sbidoul — APPROVED** [review]
Code review and test.

**Review by lmignon — APPROVED** [review]

**Pull request merged**